test/parallel/test-assert.js

```diff
@@ -1015,10 +1023,43 @@ common.expectsError(
   {
     code: 'ERR_ASSERTION',
     type: assert.AssertionError,
-    message: 'test'
+    message: 'test',
+    generatedMessage: false
   }
 );
 
+// Do not try to check Node.js modules.
```

Comment:
This might deserve a separate test file since `test-assert.js` is already 1000+ lines long..

Reply:
I personally would like to keep this test together with the other simple assert tests. I am fine to move all of those to another file though and I agree that this file is a bit long.